content/work/adobe.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Adobe Fonts
+banner: Adobe Fonts
+sub: Adobe Fonts typography superpower game
```

Comment:
Why repeat "Adobe Fonts" here? On other projects it looks like our pattern is e.g.

```yaml
title: Workflow Builder
sub: Tools for data analytics
client: Aunalytics
```

So maybe something like…

```yaml
title: Typographic Superpower Game
sub: A meditation on typography
client: Adobe Fonts
```

Or the title could be `Type-Tagging Game`, or something… Just trying to use those three fields more efficiently and consistently.